test/unit/system-state.server.test.ts

```diff
@@ -51,6 +51,9 @@ describe('getIsReconciliationNeeded()', () => {
     await prisma.systemState.deleteMany().catch(() => {});
     let result = await getIsReconciliationNeeded();
     expect(result).toBe(true);
+
+    // Restore state
+    await initialize();
```

Comment:
I would remove this, because we immediately delete the state after this test ends

Reply:
Yeah, you are right. I guess whoever wants to add tests after this can figure it out when they read the code.